src/main/java/edu/cornell/kfs/paymentworks/batch/PaymentWorksRetrieveNewVendorStep.java

```diff
@@ -90,21 +76,22 @@ public boolean execute(String jobName, Date jobRunDate) throws InterruptedExcept
 						processVendor(paymentWorksNewVendor, PaymentWorksConstants.PaymentWorksNewVendorStatus.REJECTED,
 								PaymentWorksConstants.PaymentWorksStatusText.REJECTED,
 								PaymentWorksConstants.ProcessStatus.VENDOR_REJECTED, routed);
+						jobCompletedSuccessfully = false;
 					}
-
+					LOG.info("paymentWorksNewVendor.isCustomFieldConversionErrors()" + paymentWorksNewVendor.isCustomFieldConversionErrors());
+					jobCompletedSuccessfully = jobCompletedSuccessfully && !paymentWorksNewVendor.isCustomFieldConversionErrors();
 				} catch (Exception e) {
 					LOG.error("Error processing new vendor(" + newVendor.getId() + "): " + e.getMessage());
 					routed = false;
+					jobCompletedSuccessfully = false;
 					GlobalVariables.getMessageMap().clearErrorMessages();
 				}
 			}
 		}
-
-		// send email summary
-		paymentWorksKfsService.sendSummaryEmail(writePaymentWorksNewVendorSummaryReport(newVendorSummary),
-				"PaymentWorks New Vendor Summary Report");
-
-		return true;
+		if (!jobCompletedSuccessfully) {
+			throw new RuntimeCacheException("There was an error processing new vendors from Payment Works");
```

Comment:
RuntimeException is I believe the general standard. I'm not sure what RuntimeCacheException is - haven't used that one before.